auth-web/src/components/auth/account-deactivate/DeactivateCard.vue

```diff
@@ -12,44 +12,44 @@
           >
         </div>
         <div class="ml-3 mt-1">
-          <h4 class="font-weight-bold">{{ $t(item.title) }}</h4>
-          <p>{{ $t(item.description) }}</p>
+          <h4 class="font-weight-bold">{{ t(item.title) }}</h4>
+          <p>{{ t(item.description) }}</p>
         </div>
       </div>
     </v-card-text>
   </v-card>
 </template>
 
 <script lang="ts">
-import { Component, Prop } from 'vue-property-decorator'
+import { computed, defineComponent } from '@vue/composition-api'
+import { Account } from '@/util/constants'
 import Vue from 'vue'
+import { useI18n } from 'vue-i18n-bridge'
 
-@Component({
-})
-export default class DeactivateCard extends Vue {
-  @Prop() private type: string
-
-  // no type matches means show it for all types
-  private infoArray: { title: string, description?: string, type?: string }[] = [
-    {
-      title: 'deactivateMemberRemovalTitle',
-      description: 'deactivateMemberRemovalDesc'
-    },
-    {
-      title: 'businessRemovalTitle',
-      description: 'businessRemovalDesc'
-    },
-    {
-      title: 'padRemovalTitle',
-      type: 'PREMIUM'
-    }
+export default defineComponent({
+  name: 'DeactivateCard',
+  props: {
+    type: () => Account
```

Comment:
bah sorry I know I suggested this but this is actually how you want to type this:
`type: String as PropType<Account>`